sources/install/package_base.sh

```diff
@@ -203,13 +212,15 @@ function install_fzf() {
     git -C /opt/tools clone --depth 1 https://github.com/junegunn/fzf.git
     yes|/opt/tools/fzf/install
     add-aliases fzf
+    local version
+    version=$(fzf --version | awk '{print $1}')
```

Comment:
Not writing this in one line is a mess...........
Maybe somethink like `local version; version=$(fzf --version | awk '{print $1}')` ?
See https://unix.stackexchange.com/q/687854